cloud_config/.env.app.cloud

```diff
@@ -0,0 +1,6 @@
+APP_IDENTITY_DOMAIN=matatika.eu.auth0.com
```

Comment:
this should be identity.matatika.com (probably means its wrong in CE too)

Reply:
yes, identity.matatika.com is an alias of matatika.eu.auth0.com